config/config.go

```diff
@@ -107,6 +131,18 @@ func SampleConf() *Conf {
 			},
 		},
 	}
+	cfg.Queue = QueueConfig{
```

Comment:
@placer14 when you have some cycles can you drop some knowledge here:

 This change appends the following to the daemon config:
```toml

[Queue]
  [Queue.Asynq]
    [Queue.Asynq.Asynq1]
      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_NETWORK
      #Network = "tcp"

      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_ADDR
      #Addr = "127.0.0.1:6379"

      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_USERNAME
      #Username = "default"

      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_PASSWORD
      #Password = "password"

      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_DB
      #DB = 0

      # env var: LOTUS_QUEUE_ASYNQ_ASYNQ1_POOLSIZE
      #PoolSize = 0
```
Will changes be needed here: https://github.com/filecoin-project/helm-charts/blob/master/charts/lily/values.yaml#L86 in order to deploy this? Or can I just append to it in a custom values file?

Reply:
> Yeah, I would add those values there, and then add logic in https://github.com/filecoin-project/helm-charts/blob/master/charts/lily/templates/configmap-daemonconfig.yaml which includes those values in the appropriate places

How would I go about making a deployment with these changes? They aren't ready to land in master as this is still explorative work - Is there a way to use helm charts from a feature branch?

> I'd avoid unique naming in configurations

Agreed. I don't intend for this to land in master as is - it's just a draft for exploring some ideas.  I've opted to make the config names specific to the worker back-end since other libraries (like  https://github.com/RichardKnop/machinery mentioned in the description) will more than likely require different configuration parameters.

Reply:
So these changes would be put in the next version (say `v0.8.0-rc1`) which you could build and use that for a single deployment (without using it or changing it for other deployments). Existing deployments will continue to use the chart it's configured with until we update it.

1. Update helm-charts in your dev branch.
2. `helm package ./charts/lily`
3. `helm install/upgrade <releasename> <path_to_new_lily_chart> -f <path_to_custom_values>`
4. ???
5. Profit.